src/pages/accounting/AccountCard.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react'
+
+import { Theme, createStyles, makeStyles } from '@material-ui/core/styles';
+import { Card, CardContent, Typography, Grid, Button } from '@material-ui/core/';
+import { Skeleton } from '@material-ui/lab';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    root: {
+      display: 'flex',
+    },
+    details: {
+      display: 'flex',
+    },
+    address: {
+        color: 'grey',
+        fontWeight: 400,
+      },
+    content: {
+        flexGrow: 1,
+    },
+    status: {
+        color: '#fff',
+        backgroundColor: '#76a9fa',
+    }
+  }),
+);  
+
+interface ChequebookAddress {
+    chequebookaddress: string,
+}
+
+interface ChequebookBalance {
+    totalBalance: number,
+    availableBalance: number
+}
+
+interface IProps{
+    chequebookAddress: ChequebookAddress,
+    loadingChequebookAddress: boolean,
+    chequebookBalance: ChequebookBalance,
+    loadingChequebookBalance: boolean,
+}
+
+const ConvertBalanceToBZZ = (amount: number) =>  {
+    return amount / 10000000000000000
```

Comment:
Quite magic constant, improving readability a bit :)

Also it's not a problem here since it is (so far) not used to make any transactions, but sadly number type is not precise. You might want to consider `BigInt` or if you need decimals then the BigNumber library
```suggestion
    return amount / (10 ^ 16)
```